### DISCUSSION TITLE
Questions about refactoring standard templates

### DISCUSSION THREAD

**Discussion by HavenDV** [unoplatform/uno#9166] [General]
While updating projects to 4.4.5, I noticed differences from my base projects that seem superfluous. I can provide a PR for standard templates on points you approve of:
1. Remove empty Settings and Resources from Skia.WPF projects.
2. Remove AssemblyInfo.cs from Skia.WPF projects, replacing them with:
```xml
  <ItemGroup Label="AssemblyInfo">
    <AssemblyAttribute Include="System.Runtime.InteropServices.ComVisibleAttribute">
      <_Parameter1>false</_Parameter1>
    </AssemblyAttribute>
    <AssemblyAttribute Include="System.Windows.ThemeInfo">
      <_Parameter1>System.Windows.ResourceDictionaryLocation.None</_Parameter1>
      <_Parameter1_IsLiteral>true</_Parameter1_IsLiteral>
      <_Parameter2>System.Windows.ResourceDictionaryLocation.SourceAssembly</_Parameter2>
      <_Parameter2_IsLiteral>true</_Parameter2_IsLiteral>
    </AssemblyAttribute>
  </ItemGroup>
```
3. Remove packages that are transient dependencies of other packages, eg Microsoft.Extensions.Logging may be removed because it is present in Microsoft.Extensions.Logging.Console.
4. Remove manifest files because they contain default values. The user can easily add it again if needed, VS generates a new one automatically.
5. Remove app.config.
6. Remove <PackageReference Include="Microsoft.Windows.Compatibility" Version="6.0.0" /> because it is automatically added by Uno.Wasm.Bootstrap package: https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/80ea052dc5ab37df5d5888c11c6a915917ae276c/src/Uno.Wasm.Bootstrap/build/Uno.Wasm.Bootstrap.targets#L95
7. You might also consider shipping fonts automatically with platform packages.

**Comment by jeromelaban** [comment]
Sure! Most of those make sense, except Point 6 which is going to be removed starting from .NET 7, and Point 7 which is going to be worked on soon by @agneszitte-nventive and @carldebilly. You can make individual PRs to easy the tracking. Thanks!
